layers/+lang/markdown/README.org

```diff
@@ -153,6 +161,12 @@ To generate a table of contents type on top of the buffer:
 |-------------+---------------------|
 | ~SPC m k~   | kill thing at point |
 
+** Grip-mode
+
+| Key binding | Description                                                 |
+|-------------+-------------------------------------------------------------|
+| ~SPC g p~ | toggle github flavored mb/org buffer preview in web-browser |
```

Comment:
I don't think this is necessary. Let user call grip-mode via M-x.

Reply:
I do -- I don't wanna do it -- why not have a leader key

Reply:
SPC leader key is too precious.

Reply:
I would prefer to 'more actively' inform users about this functionality by adding a keybinding for it. But why not leave it just under "ghp" as "gh" already exists and grip is 'more or less' github specific (users can still also call via `M-x` of course)

Reply:
@dalanicolai 

nope it's not GitHub specific.
GFM is widely used and is one of top makrdown implementations.

though grip makes it sounds like it's used for GitHub only, this package can be used for previewing all GFM.